### PR TITLE
Interactive list

### PR DESCRIPTION
**Pull request by elmar8287**
- Add complete status
- Add event listener to the checkbox (change).
- Store the updated array of items in local storage, so the user gets the correct list values after the page reloads

### CODE REVIEW

File: dist/index.js [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+
+function updateStatus(event, status) {
+  event.target.nextElementSibling.classList.toggle('completed');
+  status[event.target.dataset.id].completed = event.target.checked;
+  return status;
+}
+function editTodo(event, editable) {
+  editable[event.target.dataset.index].description = event.target.value;
+  return editable;
+}
```
Comment (hemantso) [inline dist/index.js:154]: - [x] As per the learning objective, We have to use proper ES6 syntax. Kindly try to use the arrow function as per ES6 guidelines.

File: dist/index.js [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+/* eslint-anable */
+
+let todoItemElems = [];
+
+function Task(description) { // this is a constructor for each task
+  this.description = description; // this comes from input
+  this.completed = false; // by default the task will be not completed
+  this.index = tasks.length + 1; // the index of inputed task
+}
+/* eslint-disable */
```
Comment (hemantso) [inline dist/index.js:17]: - [x] As per the learning objective, We have to use proper ES6 syntax. Kindly try to use the arrow function as per ES6 guidelines.

- [x] Kindly try to delete the comments from the file.

File: dist/index.js [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+    e.target.innerHTML = HTMLContent;
+    e.target.querySelector('input').checked = (checkboxStatus === 'true');
+  }
+
+}
+
+function init() {
+  let dnd = new DND(); 
+  dnd.setEventListeners();  
+}
```
Comment (hemantso) [inline dist/index.js:134]: - [x] As per the learning objective, We have to use proper ES6 syntax. Kindly try to use the arrow function as per ES6 guidelines.

File: src/status.js [hunk 4]
```diff
@@ -0,0 +1,9 @@
+export function updateStatus(event, status) {
+  event.target.nextElementSibling.classList.toggle('completed');
+  status[event.target.dataset.id].completed = event.target.checked;
+  return status;
+}
+export function editTodo(event, editable) {
+  editable[event.target.dataset.index].description = event.target.value;
+  return editable;
+}
```
Comment (hemantso) [inline src/status.js:9]: - [x] As per the learning objective, We have to use proper ES6 syntax. Kindly try to use the arrow function as per ES6 guidelines.

**Review by hemantso — CHANGES_REQUESTED** [review]
Hi @elmar8287 , 

Good job so far! 👍 
There are some issues that you still need to work on to go to the next project but you are almost there! 

Alt Text
## Required Changes ♻️
_Check the comments under the review._

Cheers and happy coding 😃

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. You can mention me (@hemant-soni-vst-au4) so I can see your comments and respond.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: dist/index.js (hunk 1 above)
Comment (elmar8287) [reply dist/index.js:154]: fixed

File: dist/index.js (hunk 3 above)
Comment (elmar8287) [reply dist/index.js:134]: fixed

File: src/status.js (hunk 4 above)
Comment (elmar8287) [reply src/status.js:9]: fixed

**Review by ZawadiSumba66 — COMMENTED** [review]
Hi @elmar8287 👋 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**